tests/data/types/generic/md_metadata.xml

```diff
@@ -0,0 +1,798 @@
+          </gmd:CI_Contact>
+        </gmd:contactInfo>
+        <gmd:role>
+          <gmd:CI_RoleCode codeList="http://standards.iso.org/iso/19139/resources/gmxCodelists.xml#CI_RoleCode" codeListValue="pointOfContact">pointOfContact</gmd:CI_RoleCode>
+        </gmd:role>
+      </gmd:CI_ResponsibleParty>
+    </gmd:contact>
+    <gmd:dateStamp>
+      <gco:Date xmlns:gco="http://www.isotc211.org/2005/gco" xsi:nil="false">2023-04-19</gco:Date>
+    </gmd:dateStamp>
+    <gmd:metadataStandardName>
+      <gco:CharacterString xmlns:gco="http://www.isotc211.org/2005/gco">ISO 19115/2003/Cor.1:2006/INSPIRE-TG2.0</gco:CharacterString>
+    </gmd:metadataStandardName>
+    <gmd:metadataStandardVersion>
+      <gco:CharacterString xmlns:gco="http://www.isotc211.org/2005/gco">GDI-Vlaanderen Best Practices - versie 2.0</gco:CharacterString>
+    </gmd:metadataStandardVersion>
+    <gmd:referenceSystemInfo>
+      <gmd:MD_ReferenceSystem>
+        <gmd:referenceSystemIdentifier>
+          <gmd:RS_Identifier>
+            <gmd:code>
+              <gmx:Anchor xmlns:gmx="http://www.isotc211.org/2005/gmx" xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.opengis.net/def/crs/EPSG/0/31370">EPSG:31370</gmx:Anchor>
+            </gmd:code>
+          </gmd:RS_Identifier>
+        </gmd:referenceSystemIdentifier>
+      </gmd:MD_ReferenceSystem>
+    </gmd:referenceSystemInfo>
+    <gmd:identificationInfo>
+      <gmd:MD_DataIdentification id="be-dov-fa3f06d1-bc45-4d06-a1e9-da79d5035537">
+        <gmd:citation>
+          <gmd:CI_Citation>
+            <gmd:title>
+              <gco:CharacterString xmlns:gco="http://www.isotc211.org/2005/gco">Opdrachten</gco:CharacterString>
+            </gmd:title>
+            <gmd:date>
+              <gmd:CI_Date>
+                <gmd:date>
+                  <gco:Date xmlns:gco="http://www.isotc211.org/2005/gco" xsi:nil="false">2023-04-19</gco:Date>
+                </gmd:date>
+                <gmd:dateType>
+                  <gmd:CI_DateTypeCode codeList="http://standards.iso.org/iso/19139/resources/gmxCodelists.xml#CI_DateTypeCode" codeListValue="revision">revision</gmd:CI_DateTypeCode>
+                </gmd:dateType>
+              </gmd:CI_Date>
+            </gmd:date>
+            <gmd:identifier>
+              <gmd:MD_Identifier>
+                <gmd:code>
+                  <gco:CharacterString xmlns:gco="http://www.isotc211.org/2005/gco">https://www.dov.vlaanderen.be/dataset/fa3f06d1-bc45-4d06-a1e9-da79d5035537</gco:CharacterString>
+                </gmd:code>
+              </gmd:MD_Identifier>
+            </gmd:identifier>
+          </gmd:CI_Citation>
+        </gmd:citation>
+        <gmd:abstract>
+          <gco:CharacterString xmlns:gco="http://www.isotc211.org/2005/gco">Een DOV-opdracht wordt meestal gebruikt om gegevens van dezelfde herkomst te groeperen in een herkenbaar geheel, Deze opdrachten vormen een snelle toegang naar een specifieke (deel)dataset. 

```

Comment:
typo
Waar wordt deze xml gegenereerd of wat is de bron van deze xml?

update_file('types/generic/md_metadata.xml',
                build_dov_url(
                    'geonetwork/srv/dut/csw'
                    '?Service=CSW&Request=GetRecordById&Version=2.0.2'
                    '&outputSchema=http://www.isotc211.org/2005/gmd'
                    '&elementSetName=full&id=8a07f330-3900-4086-89c0-'
                    'cebf940156e5'))

Reply:
This is the XML representation of the metadata of the layer dov-pub:Opdrachten. It is downloaded from Geonetwork (web version is [here](https://dov.vlaanderen.be/geonetwork/srv/eng/catalog.search#/metadata/8a07f330-3900-4086-89c0-cebf940156e5)), but Geonetwork harvests this from our Geoserver.

This needs to be updated in the Geoserver work environment and then the (metadata of the) layer dov-pub:Opdrachten needs to be republished.